spatialprofilingtoolbox/cggnn/scripts/run_sql.py

```diff
@@ -128,6 +134,20 @@ def parse_arguments():
     return parser.parse_args()
 
 
+def retrieve_importances() -> DataFrame:
+    filename = join('out', 'importances.csv')
+    return read_csv(filename, index_col=0)
```

Comment:
Should this feature be available in the DataFrame SPT workflow? That should be the preferred cggnn use option.

Reply:
You mean the functionality that pulls out DataFrames from the database? (`FeatureMatrixExtractor`)

I'm not sure, because that functionality is agnostic to files or filenames.

Reply:
But yes I copied that filename from `cggnn` source, there is probably a better way like cggnn exposing this filename. In case filename business is what you are drawing attention to.

Reply:
I mean that this PR as is does not implement importance uploading if you run the SQL-less workflow.

Reply:
So by `DataFrame SPT workflow` you mean `DataFrame (as opposed to SQL) SPT cggnn workfow`?

Yes, in the SQL-less workflow there is no importance score uploading I suppose. And that is expected behavior? Since there is nowhere for these scores to go.

Reply:
Resolved in live conversation. `run` and not `run_sql` is the preferred workflow.